docs/install-manual.md

```diff
@@ -27,7 +27,7 @@ You can run Docker Swarm on Linux 64-bit architectures. You can also install and
 
 Take a moment and identify the systems on your network that you intend to use. Ensure each node meets the requirements listed above.
 
-
+> **Warning**: Before going any further with the setup process, make sure that Docker is not pre-installed on your VM image and/or that the machine is not created using VM cloning. Docker generates a unique ID for the Engine that is located in the `/etc/docker/key.json` file. If a VM is the result of cloning with a docker daemon pre-installed, Swarm will not be able to recognize remote docker Engines as different. This is because the same unique ID is duplicated on those Engines from the VM cloning. As a result, you will see a single docker Engine registered on Swarm. As a workaround you can stop the daemon on those machines, delete the `/etc/docker/key.json` and finally restart the Engines. This will regenerate a new ID for each of those docker daemons.
```

Comment:
@MHBauer I don't actually know this particular issue.  Tho, I am close to the issue:

* VM cloning --- how would a user go about that --- it would be helpful to explain it.  Clone an image with AWS or something different?

* Does a user see a warning / error message during the setup about this?  Presenting that message will help them to debug the problem.

Also, this **Warning** really is too long to be a warning.